_summaries/mildenhall2020nerf_1.md

```diff
@@ -1,19 +1,74 @@
 ---
 layout: summary
 title: Summary
-paper: {{paper_tag}}
+paper: martin-brualla2020nerf
 # Please fill out info below
-author: # Your GitHub id
+author: mattkelleher
 score: # How did you like this paper 0(dislike) to 10(love)
 ---
 
-TODO: Summarize the paper:
-* What is the core idea?
-* How is it realized (technically)?
-* How well does the paper perform?
-* What interesting variants are explored?
+## Core Idea
+This paper introduces a fully connected deep network that synthesizes 2D views of complex 3D scenes given a location $$(x,y,z)$$ and viewing direction $$(\Theta , \Phi )$$. 
+The scene is represented by the network and is trained by using multiple images from know location and viewing directions.
+
+## Technical Implementation
+
+The 3D scenes are represented by the network as a Neural Radiance Fields (NeRF) consisting of a color and a volume density, $$\sigma$$. 
+Volume density can be thought of as the probability that a ray terminates at a given point.
+ 
+<img src="mildenhall2020nerf_1_rendering.PNG" />
+
+The network is restricted so that the location density, $$\sigma$$ is dependant only of the location while color value is depenant on location and viewing direction.  
```

Comment:
Review could have more detail about the 5D sampling, since that pretty important. But other than that, the summary is fine.